runtime/local/src/weights/pallet_dapps_staking.rs

```diff
@@ -33,41 +33,41 @@
 use frame_support::{traits::Get, weights::Weight};
 use sp_std::marker::PhantomData;
 
-/// Weight functions for pallet_dapps_staking.
+/// Weight functions for `pallet_dapps_staking`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_dapps_staking::WeightInfo for WeightInfo<T> {
 	// Storage: DappsStaking RegisteredDevelopers (r:1 w:1)
 	// Storage: DappsStaking RegisteredDapps (r:1 w:1)
 	// Storage: DappsStaking PreApprovalIsEnabled (r:1 w:0)
 	fn register() -> Weight {
-		(63_974_000 as Weight)
+		(39_675_000 as Weight)
```

Comment:
Is there a reason why the block weights changed on the local? I can see the weights are the same as https://github.com/AstarNetwork/Astar/blob/c4f078622e84121b1c3f57e49db3c262e9b40aaa/runtime/shibuya/src/weights/pallet_dapps_staking.rs#L43 and this was also executed with the `shibuya-dev` flag, but just curious why the auto-gen file was assigned to the local runtime 🤔  https://github.com/AstarNetwork/Astar/pull/567/files#diff-3949041c500f115558c821562657ee76c628c7d0c3d8e2a7a82ea34e63eff8c8R12